TempleApp/src/main/java/org/amfoss/templeapp/Activities/DeleteData.java

```diff
@@ -0,0 +1,127 @@
+package org.amfoss.templeapp.Activities;
+
+import android.app.ProgressDialog;
+import android.os.AsyncTask;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v7.app.AppCompatActivity;
+import android.util.Log;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.RadioGroup;
+import android.widget.RelativeLayout;
+import android.widget.Toast;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import org.amfoss.templeapp.JSON_API.Controller;
+import org.amfoss.templeapp.R;
+
+/**
+ * Created by Chromicle.
+ */
+public class DeleteData extends AppCompatActivity {
+
+    String id;
+    String name;
+    RadioGroup radioGroup;
+    RelativeLayout totalLayout;
+    int flag;
+    private Button delete;
+    private EditText uid1ET;
+
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.delete_data);
+        delete = (Button) findViewById(R.id.delete_btn);
+        uid1ET = (EditText) findViewById(R.id.uid);
+
+        radioGroup = (RadioGroup) findViewById(R.id.radiogroup);
+        totalLayout = (RelativeLayout) findViewById(R.id.total_View);
+
+        radioGroup.setOnCheckedChangeListener(new RadioGroup.OnCheckedChangeListener() {
+            public void onCheckedChanged(RadioGroup group, int checkedId) {
+                switch (checkedId) {
+                    case R.id.radio_donate:
+                        totalLayout.setVisibility(View.VISIBLE);
+                        flag = 0;
+                        Toast.makeText(getBaseContext(), "Selected To Delete Donated Money", Toast.LENGTH_LONG).show();
+                        break;
+                    case R.id.radio_pooja:
+                        totalLayout.setVisibility(View.VISIBLE);
+                        flag = 1;
+                        Toast.makeText(getBaseContext(), "Selected To Delete Registered Pooja", Toast.LENGTH_LONG).show();
+                        break;
+
+                }
+            }
+        });
+
+
+        delete.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+
+                if (flag == 1) {
+                    id = "REG" + uid1ET.getText().toString();
+                } else {
+                    id = "DON" + uid1ET.getText().toString();
```

Comment:
you are behind of so many commits ex: strings resources
please make sure that you are up to date with project